### PR TITLE
Enhancement / PWA updated

### PR DESCRIPTION
**Pull request by 46chris**
Had to recreate branch because I fucked up the other's commit history and the workflows weren't running

### CODE REVIEW

**Review by LJMNZS — DISMISSED** [review]
image
I see you added the manifest, but Chrome's PWA check can't seem to fetch it. 
[https://stackoverflow.com/questions/62722853/pwa-manifest-is-there-but-chrome-says-no-manifest-found](https://stackoverflow.com/questions/62722853/pwa-manifest-is-there-but-chrome-says-no-manifest-found)
This says the manifest should be able to be seen under application/manifest, but it's not showing up there, so I think you might have to register it
image
From this it looks like the manifest is missing the prefer_related_applications property

File: source/serviceWorker.js
Comment (markbussard) [inline source/serviceWorker.js:25]: So, this isn't going to work properly due to the fact that we are using parcel to compile and bundle our app. Thus, none of these files will actually exist in the build with these pathnames

Parcel has a plugin for this via: [parceljs-service-workers](https://parceljs.org/languages/javascript/#service-workers)

Just using the code snippet they provided seemed to work for me.

File: source/manifest.json
Comment (markbussard) [inline source/manifest.json:13]: When I was running this locally, I seemed to be getting errors with the src paths for these icons. It seems to work for me when removing the './' so -> img/pomatoPNG.png for example.

File: source/serviceWorker.js
Comment (markbussard) [reply source/serviceWorker.js:25]: Also, the fetch event listener will stay as is.

**Review by markbussard — CHANGES_REQUESTED** [review]

**Comment by markbussard** [comment]
Let me know if you have any questions / need help. Also, I believe we will need a separate manifests file for app and technicall done, since these are different pages and will require a different 'start_url' property?

**Review by markbussard — APPROVED** [review]
There's still a couple minor issues noted per the lighthouse report that we could add in to complete the PWA optimization criteria.

However, the app appears to work just fine in offline mode, which is the important part for the demo. So, I'd say this is good to go as long as we get some more reviews / approvals from people to ensure no obvious issues, and we can worry about the other items from the lighthouse report later.

**Pull request merged**
